### PR TITLE
Projects & About section

### PR DESCRIPTION
**Pull request by Feliverse**
### Portfolio: finish mobile version
**Learning objectives**

- Understand how to parse a Figma design to create a UI.
_(template 5)_
- Use Flexbox to place elements on the page.
- Build a personal portfolio site.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,6 +24,90 @@ <h1>Felipe Haybar</h1>
             <a class="textBoton" href="#">Start collaboration</a>
         </button>
         <div class="imagebox"></div>
-    </section>
+        <section class="projects">
```
Comment (OyePriscilla) [inline index.html]: - [x] KIndly display this main section as `grid` according to the requirement.  The previous reviewer stated this as well. 😄

File: index.html [hunk 2]
```diff
@@ -24,6 +24,90 @@ <h1>Felipe Haybar</h1>
...
+            <img src="images/project4.png" alt="project4" width="320px">
+            <div class="pData">
+                <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                <button class="see">
+                    <a class="textBoton" href="#">See this project →</a>
```
Comment (OyePriscilla) [inline index.html]: - [optional] Please add user interaction styling to all the buttons (hover, enable ...) in your project as stated in the requirement. 

b16

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Feliverse    😃 , 

Good job so far!
There are little issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
LInters checks passed ✔️ 
GitHub flow observed ✔️ 

## Required Changes ♻️
Kindly check the inline comments below for the required changes.

## Optional suggestions

- [ optional] Kindly ensure that your commit messages are in sentence case for subsequent PR's. 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Feliverse) [reply index.html]: Hi Oye, 
grid applied on the projects section and about section, look better.
Thank you

File: index.html (hunk 2 above)
Comment (Feliverse) [reply index.html]: Hi Oye,
I had to apply hover config to my buttons, nice suggestion
see you

**Comment by Feliverse** [comment]
All observation from feedbackers was resolve
Thanks to all

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @Feliverse, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Milestone worked as expected. :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**